org.nelson.Nelson.yaml

```diff
@@ -0,0 +1,314 @@
+        sha256: e4defbac30f50d64e1556374aa49e574417c9e72c6b1de7a4ff88c4b1bea6e9b
+
+  - name: eigen
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DBUILD_TESTING=OFF
+    sources:
+      - type: archive
+        url: https://gitlab.com/libeigen/eigen/-/archive/3.4.0/eigen-3.4.0.tar.gz
+        sha256: 8586084f71f9bde545ee7fa6d00288b264a2b7ac3607b974e54d13e7162c1c72
+
+  - name: portaudio
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: http://files.portaudio.com/archives/pa_stable_v190700_20210406.tgz
+        sha256: 47efbf42c77c19a05d22e627d42873e991ec0c1357219c0d74ce6a2948cb2def
+
+  - name: matio
+    buildsystem: cmake
+    sources:
+      - type: archive
+        url: https://github.com/tbeu/matio/archive/refs/tags/v1.5.26.tar.gz
+        sha256: 4aa5ac827ee49a3111f88f8d9b8ae034b8757384477e8f29cb64582c7d54e156
+    cmake-options:
+      -DBUILD_SHARED_LIBS=ON
+      -DENABLE_EXTENDED_SPARSE=ON        
+
+  - name: fftw3
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -Wno-dev # suppress warning for project developers
+      - -DENABLE_THREADS=ON
+    sources:
+      - type: archive
+        url: http://www.fftw.org/fftw-3.3.10.tar.gz
+        sha256: 56c932549852cddcfafdab3820b0200c7742675be92179e59e6215b340e26467
+
+  - name: fftw3f
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -Wno-dev # suppress warning for project developers
+      - -DENABLE_THREADS=ON
+      - -DENABLE_FLOAT=ON
+    sources:
+      - type: archive
+        url: http://www.fftw.org/fftw-3.3.10.tar.gz
+        sha256: 56c932549852cddcfafdab3820b0200c7742675be92179e59e6215b340e26467
+
+  - name: nelson
+    buildsystem: cmake
+    builddir: true
+    build-options:
+      env:
+        CPPFLAGS: "-I/app/include"
+        LDFLAGS: "-L/app/lib"
+        PKG_CONFIG_PATH: "/app/lib/pkgconfig:/app/share/pkgconfig"
+        CMAKE_PREFIX_PATH: "/app"
+    config-opts:
+      - -DCMAKE_PREFIX_PATH=/app
+      - -DEigen3_DIR=/app/share/eigen3/cmake
+      - -DMPI_ROOT=/app
+      - -DMPI_C_COMPILER=/app/bin/mpicc
+      - -DMPI_CXX_COMPILER=/app/bin/mpicxx
+      - -DBOOST_ROOT=/app
+      - -DBoost_DIR=/app/lib/cmake/Boost-1.84.0
+      - -DBLA_VENDOR=OpenBLAS
+      - -DOpenBLAS_ROOT=/app
+      - -DLAPACK_ROOT=/app
+      - -DLAPACKE_ROOT=/app      
+      - -DHDF5_ROOT=/app
+      - -DHDF5_DIR=/app/share/cmake/hdf5
+      - -DCURL_INCLUDE_DIR=/app/include
+      - -DCURL_LIBRARY=/app/lib/libcurl.so
+      - -Wno-dev
+    build-commands:
+      - cmake .. ${CMAKE_ARGS}
+      - make -j $FLATPAK_BUILDER_N_JOBS
+      - make buildhelp
+      - desktop-file-edit --set-icon=${FLATPAK_ID} ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+
+
+    sources:
+      - type: git
+        url: https://github.com/nelson-numerical-software/nelson.git
+        tag: master
```

Comment:
master isn't a tag. and never use a git branch. this should be `tag` + `commit` or just `commit`.